tests/test_fetch_nearest_dataset_names.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+from pmlb import fetch_data, fetch_nearest_dataset_names
+
+
+def test_nearest_dataset_is_itself():
+    """Tests whether the nearest dataset is itself"""
+
+    for test_dataset in ['vote', 'analcatdata_aids', 'car']:
+        df = fetch_data(test_dataset, local_cache_dir='../datasets')
```

Comment:
I wonder if we should write `local_cache_dir='datasets'` because the test will be run from root directory.

Reply:
the tests run correctly from root directory with `../datasets' as the path. i was getting an error with `local_cache_dir=datasets' ...